ui/pages/confirmations/hooks/useConfirmationNetworkInfo.test.ts

```diff
@@ -61,28 +47,6 @@ describe('useConfirmationNetworkInfo', () => {
     );
 
     expect(result.current.networkDisplayName).toBe('Custom Mainnet RPC');
-    expect(result.current.networkImageUrl).toBe('./some_image');
```

Comment:
`imageUrl` is a field I'm not sure about.  I don't see it anywhere in the `NetworkController`, yet there are places in extension expecting it in `providerConfig` and `networkConfigurations`.